content/docs/intro/cloud-providers/azure/_index.md

```diff
@@ -72,3 +72,5 @@ The Azure provider accepts the following configuration settings.  These can be p
 * `subscriptionId`: (Optional) The subscription ID to use. It can also be sourced from the `ARM_SUBSCRIPTION_ID` environment variable.
 * `tenantId`: (Optional) The tenant ID to use. It can also be sourced from the `ARM_TENANT_ID` environment variable.
 * `useMsi`: (Optional) Set to true to authenticate using managed service identity. It can also be sourced from the `ARM_USE_MSI` environment variable.
+
+For Pulumi Azure support and troubleshooting, click the links in the sidebar on the left of the page.
```

Comment:
I get the intent, but this phrasing feels inappropriate to me, as there isn't a link in the sidebar that will actually help you with "Azure support and troubleshooting" as such.

Reply:
Ah, I saw the Pulumi Support/Troubleshooting part on the sidebar and assumed that this was included in that. No worries, we can probably just cut that secondary keyword (Azure support) altogether if it's inaccurate.